pkg/storage/bloom/v1/tokenizer.go

```diff
@@ -42,8 +42,8 @@ func NewNGramTokenizer(n, skip int) *NGramTokenizer {
 
 // The Token iterator uses shared buffers for performance. The []byte returned by At()
 // is not safe for use after subsequent calls to Next()
-func (t *NGramTokenizer) Tokens(line string) NGramTokenIter {
-	return NGramTokenIter{
+func (t *NGramTokenizer) Tokens(line string) Iterator[[]byte] {
```

Comment:
Can you add a comment that this function implements the new `NGramBuilder` interface?

Reply:
Done